### PR TITLE
Move `linux_checks_librs` to own module

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Part of #329

### CODE REVIEW

**Comment by catornot** [comment]
maybe `linux_checks` gets compiled on windows too?

**Comment by GeckoEidechse** [comment]
> maybe `linux_checks` gets compiled on windows too?

Yeah but like both Linux and Windows say unused function... T_T

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -42,6 +42,8 @@ use mod_management::{
 mod northstar;
 use northstar::get_northstar_version_number;
 
+mod platform_specific;
```
Comment (catornot) [inline src-tauri/src/main.rs:45]: you had this before

```
#[cfg(target_os = "windows")]
use platform_specific::windows;


#[cfg(target_os = "linux")]
use platform_specific::linux;
```

File: src-tauri/src/lib.rs [hunk 2]
```diff
@@ -9,9 +9,6 @@ mod platform_specific;
 #[cfg(target_os = "windows")]
 use platform_specific::windows;
```
Comment (catornot) [inline src-tauri/src/lib.rs:10]: also why wasn't this moved/removed?

File: src-tauri/src/main.rs (hunk 1 above)
Comment (GeckoEidechse) [reply src-tauri/src/main.rs:45]: Nope that just gives an additional error of
```
warning: unused import: `platform_specific::linux`
  --> src/main.rs:49:5
   |
49 | use platform_specific::linux;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```
which makes like no sense to me T_T

**Pull request closed without merging**